package.json

```diff
@@ -34,7 +36,14 @@
     "@babel/preset-env": "^7.9.5",
     "@babel/preset-react": "^7.9.4",
     "babel-loader": "^8.1.0",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.1.0",
```

Comment:
I removed the hackreactor one below since we're using the airbnb eslint config. I didn't want them conflicting.

Reply:
Where is the hack reactor one?